src/Kiota.Builder/Writers/Ruby/RubyConventionService.cs

```diff
@@ -77,7 +77,7 @@ internal void AddRequestBuilderBody(CodeClass parentClass, string returnType, La
         if (parentClass.GetPropertyOfKind(CodePropertyKind.PathParameters) is CodeProperty pathParametersProp &&
             parentClass.GetPropertyOfKind(CodePropertyKind.RequestAdapter) is CodeProperty requestAdapterProp)
         {
-            var urlTemplateParams = urlTemplateVarName ?? $"@{pathParametersProp.Name.ToSnakeCase()}";
+            var urlTemplateParams = string.IsNullOrEmpty(urlTemplateVarName) ? $"@{pathParametersProp.Name.ToSnakeCase()}" : urlTemplateVarName;
```

Comment:
Safe to assume that pathParametersProp.Name is not empty?

Reply:
same case as other instance, resolving